src/System.Threading/tests/MutexTests.cs

```diff
@@ -70,11 +70,8 @@ public void Ctor_NameUsedByOtherSynchronizationPrimitive_Windows()
             }
         }
 
-        [Fact]
         [PlatformSpecific(TestPlatforms.Windows)]
-        [SkipOnTargetFramework(
-            TargetFrameworkMonikers.Uap,
-            "Impersonation APIs are not available and creating global sync objects is not allowed in UWP apps.")]
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWinRT))] // Can't create global objects in appcontainer
```

Comment:
That is true, but We know that Uap runs inside an AppContainer and UapAot doesn't. However if we have plans on running UapAot inside AppContainer then this would pick those up.